src/app/repository.ts

```diff
@@ -0,0 +1,14 @@
+export interface Repository {
+  name: string;
+  description: string | null;
```

Comment:
Have you considered making some fields optional instead of having an empty value?

Reply:
I had not considered that but I like it! updated